uptime/uptime_windows.go

```diff
@@ -6,5 +6,11 @@ var getTickCount = syscall.NewLazyDLL("kernel32.dll").NewProc("GetTickCount")
 
 func get() (float64, error) {
 	r, _, err := getTickCount.Call()
+	if errno, ok := err.(syscall.Errno); ok {
+		if errno != 0 {
```

Comment:
I think it's better to use some constants to represent `ERROR_SUCCESS`, rather than raw error code `0`. Is it possible?
(I don't know whether there's libraries to provide such constants. If not, current raw error code is enough.)